pkg/ccl/logictestccl/testdata/logic_test/partitioning_index

```diff
@@ -184,3 +184,31 @@ inverted  CREATE TABLE public.inverted (
           FAMILY "primary" (a, b, j)
 )
 -- Warning: Partitioned table with no zone configurations.
+
+# Regression test for #60019. The index predicate should be formatted after the
+# PARTITION BY clause to match the syntax that is accepted.
```

Comment:
you can add a CREATE TABLE round-trip style test after https://github.com/cockroachdb/cockroach/pull/60513 merges :P oh well

Reply:
(this is now possible!)

Reply:
I added one.